modules/azure-gaming/game-dev-vm/test/main.test.bicep

```diff
@@ -0,0 +1,41 @@
+param location string = resourceGroup().location
+param vmName string = 'bicep'
+param adminName string = 'dcibadmin'
+param vmSize string = 'Standard_D4s_v3'
+
+@secure()
+param adminPass string = newGuid()
+
+var engines = [
+  // 'no_engine'
+  'ue_4_27_2'
+  // 'ue_5_0_1'
+]
+
+@batchSize(1)
+module win10 '../main.bicep' = [for engine in engines: {
+  name: 'win10_${engine}'
+  params: {
+    location  : location
+    vmName    : vmName
+    adminName : adminName
+    adminPass : adminPass
+    osType    : 'win10'
+    gameEngine: engine
+    vmSize    : vmSize
+  }
+}]
+
+// @batchSize(1)
```

Comment:
Is the test needed? If not, can we remove it?

Reply:
@shenglol , in order for the rest of these tests to pass, I would need you to approve the rest of the VM images. 

How many images should we test out of the 6 total

Reply:
lets circle pack in a new PR though if we want to add more tests. going to clean this up for now.